base_exception/models/base_exception.py

```diff
@@ -271,19 +265,6 @@ def _compute_exceptions_summary(self):
             else:
                 rec.exceptions_summary = False
 
-    def _popup_exceptions(self):
```

Comment:
This method was used on `purchase_exception`, but it can be restored there, not here, while migrating it.

Reply:
Also in `account_move_exception`. Please, accept https://github.com/OCA/server-tools/pull/2373.